src/main/java/graphql/Scalars.java

```diff
@@ -69,7 +69,9 @@ private Integer convertImpl(Object input) {
         public Integer serialize(Object input) {
             Integer result = convertImpl(input);
             if (result == null) {
-                throw new CoercingSerializeException("Invalid value '" + input + "' for Int");
+                throw new CoercingSerializeException(
+                  "Expected type 'Int' but was '" + input.getClass().getSimpleName() + "'."
```

Comment:
Sorry once again but I think input here can be null and hence a NPE is waiting to happen.

Have a `mkType()` function and if its null return "null" otherwise this error message

Reply:
Yep, will do.